src/SpeechSynthesis.jsx

```diff
@@ -51,7 +51,9 @@ const SpeechSynthesis = (props) => {
   }, []);
 
   const speak = (args = {}) => {
-    const { voice = null, text = '', rate = 1, pitch = 1 } = args;
+    const {
+      voice = null, text = '', rate = 1, pitch = 1, volume = 0.5
```

Comment:
This looks good, but the default volume value should probably be 1 to match the underlying API when it isn't set.
https://developer.mozilla.org/en-US/docs/Web/API/SpeechSynthesisUtterance/volume

Reply:
Changed default volume to 1